packages/features/eventtypes/components/EventTypeDescription.tsx

```diff
@@ -78,11 +105,21 @@ export const EventTypeDescription = ({
               </Badge>
             </li>
           )}
-          {eventType.metadata?.managedEventConfig && !isPublic && (
+          {showManagedBadge && !showManagedByToolTip && (
             <Badge variant="gray" startIcon="lock">
               {t("managed")}
             </Badge>
           )}
+          {showManagedBadge && showManagedByToolTip && (
+            <Tooltip
+              content={numOfManagedByAdmins <= 2 ? managedByToolTipContent : managedByToolTipContentWithLink}>
+              <div>
+                <Badge variant="gray" startIcon="lock">
+                  {t("managed")}
+                </Badge>
+              </div>
+            </Tooltip>
+          )}
```

Comment:
Don't need to conditionally show the tooltip and use the same batch two times. Instead, keep the badge inside the tooltip for both conditions and disable the tooltip when the length of the admins is 0. I think there are props present to open the tooltip, so use it.

Reply:
i had tried with disabling tooltip, but there are no props for disabling. There is one `aria-disabled` but it will still show empty tooltip.

Reply:
You can use the hidden class inside the classname conditionally, but it’s not necessary.

Hey, it just came to my mind now, that in a team the owner has the highest priority. This means there must be an owner on the team, but admins might not be present. As the owner, you can assign roles to others, but as an admin, you cannot edit the owner.

So, the main point is to show the owner's name instead of the admin's name. This way, you won't need to conditionally render anything, because there will always be an owner present in the team. We only need to check the owner length to show the number if the length is greater than two, just like we do for the admin.

And you don't need to conditionally render badge also.

Apart from this all things are perfect.

Reply:
Let's go with 'admin' for now, as per the requirement.